packages/web-components/src/components/va-featured-content/va-featured-content.tsx

```diff
@@ -7,18 +7,35 @@ import { Component, Host, h } from '@stencil/core';
  */
 @Component({
   tag: 'va-featured-content',
-  styleUrl: 'va-featured-content.css',
+  styleUrl: 'va-featured-content.scss',
   shadow: true,
 })
 export class VaFeaturedContent {
+  /**
+   * Whether or not the component will use USWDS v3 styling.
+   */
+  @Prop() uswds?: boolean = false;
+
   render() {
-    return (
-      <Host>
-        <div class="feature">
-          <slot name="headline"></slot>
-          <slot />
-        </div>
-      </Host>
-    );
+    const { uswds } = this;
+    if (uswds) {
+      return (
+        <Host>
+          <div class="usa-summary-box">
```

Comment:
The [USWDS example](https://designsystem.digital.gov/components/summary-box/) has all of these on the wrapping element (which you might able to simplify by putting them on the `<Host>` element):

```
  class="usa-summary-box"
  role="region"
  aria-labelledby="summary-box-key-information"
```

And is there any way to assign the other USWDS classes? Maybe programmatically on the assigned nodes?

- usa-summary-box__body
- usa-summary-box__heading
- usa-summary-box__text
- usa-summary-box__link

```js
    this.el.shadowRoot
      .querySelector('slot')
      .assignedNodes();
```

Reply:
I was also thinking that maybe, even if you were able to assign the classes to matching elements, the styles would be blocked by the shadow dom so this might be a good candidate for `shadow: false` on the component.

Reply:
I was looking for a way to add these, thanks! I can't find a way to access the actual slot content to add `usa-summary-box__link` to the links, but was able to add the others.